components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -593,3 +657,55 @@ func updateRetryingDescription(desc string, newDesc string) string {
 
 	return desc + ", " + newDesc
 }
+
+func (m *orchestrationManager) waitForStart(o *internal.Orchestration) ([]orchestration.RuntimeOperation, error) {
+	maintenancePolicy, err := m.getMaintenancePolicy()
+	if err != nil {
+		m.log.Warnf("while getting maintenance policy: %s", err)
+	}
+
+	//polling every 5 min until ochestration start
+	pollingInterval := 5 * time.Minute
+	var operations, unnotified_operations []orchestration.RuntimeOperation
+	err = wait.PollImmediateInfinite(pollingInterval, func() (bool, error) {
+		//resolve operations, cancel non existent ones
+		operations, err = m.resolveOperations(o, maintenancePolicy)
+		if err != nil {
+			_, err = m.failOrchestration(o, fmt.Errorf("failed to resolve operations: %w", err))
```

Comment:
I think we shouldn't fail the orchestration if there is a (temporary) resolution failure

```suggestion
```